### PR TITLE
Sets docker config for rhel7-64le build

### PR DESCRIPTION
**Pull request by mahdipub**
Sets appropriate docker-image and sets compiler-cmd='' as gcc is already set in docker imagage in `linux_ppc-64_cmprssptrs_le.spec` and `linux_ppc-64_le.spec` spec files. Infrastructure issue 9271

Signed-off-by: mahdi@ibm.com

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
Depends tooling#1129

**Comment by AdamBrousseau** [comment]
Jenkins line endings check

**Review by AdamBrousseau — APPROVED** [review]
@mahdipub can you improve your commit comment a bit please?
Please refer to the contributing guidelines
https://github.com/eclipse-openj9/openj9/blob/master/CONTRIBUTING.md#commit-guidelines
Since the issue is internal, I would put something like this in the footer
`Infrastructure issue 9271`

File: buildspecs/linux_ppc-64_cmprssptrs_le.spec [hunk 1]
```diff
@@ -56,10 +56,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<property name="graph_make_parallel_arg" value="-j `numberOfCPUs`"/>
 		<property name="graph_req.arch0" value="arch:ppcle"/>
 		<property name="graph_req.arch1" value=""/>
-		<property name="graph_req.aux0" value="{$common.req.build.cuda$}"/>
+		<property name="graph_req.aux0" value="docker"/>
 		<property name="graph_req.aux1" value=""/>
-		<property name="graph_req.build" value="{$common.req.build.java9$}"/>
-		<property name="graph_req.build2" value="{$common.req.build.java8$}"/>
+		<property name="graph_req.build" value=""/>
+		<property name="graph_req.build2" value=""/>
```
Comment (AdamBrousseau) [inline buildspecs/linux_ppc-64_cmprssptrs_le.spec:62]: I don't think we should be removing the build capability here. We still use it for the other docker spec (xlinux). Without it, the job capabilities are fairly generic.

**Review by AdamBrousseau — CHANGES_REQUESTED** [review]
Add back the build capability

**Comment by AdamBrousseau** [comment]
jenkins compile plinux jdk11

File: buildspecs/linux_ppc-64_cmprssptrs_le.spec (hunk 1 above)
Comment (mahdipub) [reply buildspecs/linux_ppc-64_cmprssptrs_le.spec:62]: Done

Comment (AdamBrousseau) [reply buildspecs/linux_ppc-64_cmprssptrs_le.spec:62]: Right, but it is a bit risky and out of the normal to not use the `build:` label for a compile job. It allows us to easily see if a machine is used for compiles. We still use it for the xlinux specs so I would prefer we keep it.

**Comment by AdamBrousseau** [comment]
Jenkins compile plinux jdk17

**Review by AdamBrousseau — APPROVED** [review]

**Pull request merged**